packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -485,11 +504,300 @@ describe('postgresql', () => {
     expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(`
 
 
-            - Introspecting based on datasource defined in prisma/using-dotenv.prisma
+                              - Introspecting based on datasource defined in prisma/using-dotenv.prisma
 
-            ✖ Introspecting based on datasource defined in prisma/using-dotenv.prisma
+                              ✖ Introspecting based on datasource defined in prisma/using-dotenv.prisma
 
-        `)
+                                                    `)
+    expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+  })
+})
+
+describeIf(!process.env.TEST_SKIP_MSSQL)('sqlserver-multi-schema', () => {
+  if (process.env.CI) {
+    // to avoid timeouts on macOS
+    jest.setTimeout(80_000)
+  } else {
+    jest.setTimeout(20_000)
+  }
+
+  const connectionString = process.env.TEST_MSSQL_URI || 'mssql://SA:Pr1sm4_Pr1sm4@localhost:1433/master'
```

Comment:
Is the second branch supposed to persist on main, or just something for local testing?

Reply:
I think we should get rid of the fallbacks, some are still around but not needed since the local setup relies on `.envrc`

Reply:
We could do that in a separate PR